src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,44 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String color;
```

Comment:
Don't forget about access modifiers for your custom fields. It's a good practice to make fields private and provide public getters and setters for access.